routes/comments.js

```diff
@@ -63,11 +70,11 @@ module.exports = function () {
 
 
 
-    router.post('/create_comment', function (req, res) {
+    router.post('/create_comment', require_signin, function (req, res) {
         var mysql = req.app.get('mysql');
         var sql = "INSERT INTO comments (post_id, user_id, content, comment_date) VALUES (?,?,?,?)";
         var date = new Date();
-        var inserts = [req.body.post_id, req.body.user_id, req.body.content, date];
+        var inserts = [req.session.user_id, req.body.user_id, req.body.content, date];

```

Comment:
You likely meant to put "req.session.user_id" as the second argument, the first argument is the id of the original post in which to leave a comment.

Reply:
That is correct! 
var inserts = [req.body.post_id, req.session.user_id, req.body.content, date];